helix-lsp/src/client.rs

```diff
@@ -318,6 +318,17 @@ impl Client {
                     inlay_hint: Some(lsp::InlayHintWorkspaceClientCapabilities {
                         refresh_support: Some(false),
                     }),
+                    workspace_edit: Some(lsp::WorkspaceEditClientCapabilities {
+                        document_changes: Some(true),
+                        resource_operations: Some(vec![
+                            lsp::ResourceOperationKind::Create,
```

Comment:
I don't think we actually handle these, do we? Should we not advertise these capabilities?

Reply:
we do actually but it seems we just forgot to add the capability (many LS were sending these regardless :D):

https://github.com/helix-editor/helix/blob/14f1a62367fe70456201c0a64174b643cf294fdd/helix-term/src/commands/lsp.rs#L860

Reply:
Ahh ok cool